hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -273,6 +358,15 @@ def _check_company_id_add_line_task_id(self):
                     _('The Company in the Timesheet Sheet and in '
                       'the Task must be the same.'))
 
+    @api.multi
+    def _get_possible_reviewers(self):
+        self.ensure_one()
+        ResUsers = self.env['res.users']
+        result = ResUsers
+        if self.review_policy == 'hr':
+            result |= self.env.ref('hr.group_hr_user').users
```

Comment:
I don't know if it will give result in domain of the company of sheet

Reply:
Sorry, didn't get this one

Reply:
my assumption is
sheet belongs to one company, self.env.ref('hr.group_hr_user') will return users from all companies so in result you may get reviewer which not belongs to this company,

Reply:
Oh, got it. Great question! Seems yes? https://github.com/odoo/odoo/blob/c789426076c6f2ef15d2f93722fad78088d3120c/odoo/addons/base/models/ir_model.py#L1647

@mreficent @pedrobaeza maybe you can give a hint?

Reply:
Yes, it's filtered by current company domain as it's not done with sudo, so it shouldn't be any problem.